### PR TITLE
[WEBSITE] lighthouse audit fixes

### LINKED ISSUE #3154: Close button icons do not have an accessible name
## Description
When using VoiceOver/Safari/iOS 12.1.4 and swiping to the "X"  icon in the .top right corner of  the "we are hiring"  container, the user hears this control announced as the word "button".  Screen reader users will not know what this button does without providing an accessible name.

In the HTML for this button below, we can see that there is no text or attribute for this button to have an accessible name.

`<button _ngcontent-c0="" class="close" type="button"><clr-icon _ngcontent-c0="" shape="close"><svg version="1.1" viewBox="0 0 36 36" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" aria-hidden="true" role="img"><path class="clr-i-outline clr-i-outline-path-1" d="M19.41,18l8.29-8.29a1,1,0,0,0-1.41-1.41L18,16.59,9.71,8.29A1,1,0,0,0,8.29,9.71L16.59,18,8.29,26.29a1,1,0,1,0,1.41,1.41L18,19.41l8.29,8.29a1,1,0,0,0,1.41-1.41Z"></path></svg></clr-icon></button>`

## Location
Mobile iOS home page

## Priority
High

## Recommended fix
I recommend using aria-label to add an accessible name for this button as seen below.

`<button aria-label="close we are hiring" class="header-hamburger-trigger" type="button"><span></span></button>`

## LOE
fast

### PR DESCRIPTION
**Pull request by coryrylan**
Multiple small fixes for the documentation site related to SEO and a11y.

closes #3154 #3155 #3172 #3184

### CODE REVIEW

**Comment by gnomeontherun** [comment]
The icons you added have a shadow, is that intentional?

**Comment by coryrylan** [comment]
> The icons you added have a shadow, is that intentional?

Good catch, it wasn't intentional. I didnt notice the shadow in VSCode when looking at the images. I'll update it.

**Comment by coryrylan** [comment]
Icons fixed

File: src/website/src/app/utils/external-link.directive.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, ElementRef, HostBinding, PLATFORM_ID, Inject } from '@angular/core';
+import { isPlatformBrowser } from '@angular/common';
+
+@Directive({
```
Comment (gnomeontherun) [inline src/website/src/app/utils/external-link.directive.ts]: We should consider this for our a11y toolkit.

File: src/website/src/manifest.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Clarity Design",
+  "short_name": "Clarity",
+  "theme_color": "#0065ab",
+  "background_color": "#ffffff",
+  "display": "standalone",
+  "Scope": "/",
```
Comment (gnomeontherun) [inline src/website/src/manifest.json]: Caps, is that an issue?

Comment (coryrylan) [reply src/website/src/manifest.json]: Hmm it was generated by a tool but I will double check to see what the spec is.

File: src/website/src/app/utils/external-link.directive.ts (hunk 1 above)
Comment (coryrylan) [reply src/website/src/app/utils/external-link.directive.ts]: Is this something that already exists? If so where?

**Pull request merged**